### DISCUSSION TITLE
OpenGL Renderer Crash

### DISCUSSION THREAD

**Discussion by fantasyzhjk** [ppy/osu#30157] [Q&A]
### Type

Crash to desktop

### Bug description

random crashes while playing (maybe fullscreen only)

### Screenshots or videos

![image](https://github.com/user-attachments/assets/31ee476e-7037-41d1-94e1-8bb7c9aaa793)

### Version

2024.906.2

### Logs

[1728446149.runtime.log](https://github.com/user-attachments/files/17301787/1728446149.runtime.log)
[1728446149.performance.log](https://github.com/user-attachments/files/17301788/1728446149.performance.log)
[1728446149.network.log](https://github.com/user-attachments/files/17301789/1728446149.network.log)
[1728446149.auth.log](https://github.com/user-attachments/files/17301820/1728446149.auth.log)
[crashlog from windows event viewer.txt](https://github.com/user-attachments/files/17301798/crashlog.from.windows.event.viewer.txt)

**Comment by bdach** [comment]
Please begin by checking for drivers updates for your graphics card which may resolve this.

**Reply by fantasyzhjk** [reply]
I updated the nvidia driver to 565.90 and also osu to 2024.1009.0, but the problem still exists

I also noticed that when I alt-tab back to osu! from another application, it freezes for about a second.

heres my system specs:

- Windows 11 10.0.26100.1742 (24H2)
- AMD Ryzen 5 3600X
- NVIDIA GeForce RTX 2060 6G

**Reply by bdach** [reply]
In that case I can't really provide further counsel, unless you provide something like [traces from gpuview](https://nvidia.custhelp.com/app/answers/detail/a_id/3507/).

If you decide trying this, **do not post the resulting file publically**. You can e-mail it to my publically-listed github address instead.

**Reply by bdach** [reply]
correct, it's that email address. i'm deleting your message because otherwise that email will be listed to anonymous users and i will get spam.

**Reply by fantasyzhjk** [reply]
uhhh, after a few crashes, it stopped crashing.
but im still going to send you the traces...
theres another issue with directx though. This one might be easier to fix?

**Reply by bdach** [reply]
So the traces you sent contain no crashes? That's not useful to me sorry.

And as for the directx issue - it depends what the issue is?

**Reply by fantasyzhjk** [reply]
ok so the trace I sent you is about the directx issue
the game crashes when I switch on/off fullscreen
you can check the runtime log that I included in the email

**Reply by bdach** [reply]
Right, okay.

I looked at the trace but I don't see much to investigate there either, not an error message in sight. I've seen that sort of stuff before and wasn't able to diagnose any better (https://github.com/ppy/osu/discussions/25754). About the only trace that connects the two reports is that it's an nvidia card.

Maybe you can try disabling something like shadowplay if you have such things enabled? Really not sure, this is the sort of thing that is almost impossible to debug without having access to the hardware.

**Reply by fantasyzhjk** [reply]
I successfully reproduced the OpenGL issue today
send you a new trace later?
